Cargo.toml

```diff
@@ -44,19 +45,14 @@ webpki = "~0.21.3"
   [dependencies.igd]
   version = "~0.11.1"
   features = [ "aio" ]
-  optional = true
 
   [dependencies.rustls]
   version = "~0.17.0"
   features = [ "dangerous_configuration" ]
 
 [dev-dependencies]
-clap = "2.33.1"
-tracing = "~0.1.15"
+async-std = "1.6.5"
```

Comment:
Why do you need `async-std`? Can't you just use tokio? Note: I'm not saying tokio is better than async-std (or vice-versa), just that it seems unnecessary to bring in another runtime when we already have tokio.